target-platform/target-platform.target

```diff
@@ -45,6 +45,10 @@
 			<repository location="https://download.eclipse.org/cbi/updates/license/"/>
 			<unit id="org.eclipse.license.feature.group" version="0.0.0"/>
 		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://download.eclipse.org/technology/swtbot/releases/4.2.1/"/>
```

Comment:
A lot of the dependencies are part of the release train. Wouldn't it be cleaner to consume them via https://download.eclipse.org/releases/2024-06, rather than collecting everything over 6 locations?

Reply:
This indeed seems cumbersome but is intentionally, because m2e contributes to the SimRel and the repository you mentioned is the SimRel aggregation.
And since we want to make sure that there are no self-loops in the SimRel we don't use that repository.